docker/dev/ubuntu/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Develop Orion Context Broker Ubuntu-Latest
+FROM ubuntu:latest
```

Comment:
I agree in that we should provide also CentOS 6.x (it would be a bit weird to include Ubuntu, which is not official, before including the official one). How complex is to create a docker file for CentOS 6.x?